src/xjs/parser.ts

```diff
@@ -320,7 +320,6 @@ export async function parse(tpl: string, filePath = "", lineOffset = 0, columnOf
             } else if (lookup(V_RW)) {
                 advance(V_RW);
                 nd.defaultValue = currentText();
-                console.log("TODO: support default value as variable: ", nd.defaultValue);
```

Comment:
... and xjs changes should be submitted on the xjs repo :-) (I will also share it with you)